client_test.go

```diff
@@ -1,19 +1,39 @@
 package iex
 
 import (
+	"io"
 	"net/http"
+	"net/http/httptest"
 	"testing"
 	"time"
 )
 
```

Comment:
Lowercase maybe? It doesn't really matter since `*_test.go` files are only compiled when running tests, but I feel like lowercase would more align with the fact that this is a local testing struct.